src/game/game.js

```diff
@@ -413,7 +413,7 @@
             });
 
             if(!runCodeCache[userId].globals._) {
-                runCodeCache[userId].globals._ = _.runInContext();
+                runCodeCache[userId].globals._ = _.runInContext(runCodeCache[userId].globals);
```

Comment:
That's a very good point. The entire lodash module needs to be loaded in the slave context. I've taken this change out for now.